### DISCUSSION TITLE
局域网wifi设备无法发送文件到有线链接下的pc设备。

### DISCUSSION THREAD

**Discussion by FIlN** [localsend/localsend#1077] [L-简体中文]
### 问题表现
1. 水星路由器一台，wifi ap隔离都已关闭。
2. 笔记本电脑、手机各一台，链接wifi网络，手机与笔记本正常收发文件。
3. 台式机通过网线连接到水星路由器。
4. 台式机可以向笔记本发送文件，但是笔记本向台式机发送文件报错：

> DioException [connection error]: The connection errored: 信号灯超时时间已到
 This indicates an error which most likely cannot be solved by the library.
Error: SocketException: 信号灯超时时间已到
 (OS Error: 信号灯超时时间已到
, errno = 121), address = 192.168.0.106, port = 8316

5. 按照软件故障排除提示，排查防火墙53317端口已经允许udp和tcp传入
![1704591500223](https://github.com/localsend/localsend/assets/21956412/434afdfc-d067-4cde-ad79-c4c3115c51fb)

**Comment by Alexander4731** [comment]
遇到了同样的问题，由于我的台式机有网卡，确认切换到wifi连接没问题。
请问楼主解决了嘛

**Comment by CrazyMayfly** [comment]
或许可以尝试一下我的文件传输工具，看看是不是协议的问题
https://github.com/CrazyMayfly/File-Transfer-Tools

**Comment by chengdidididi** [comment]
我也遇到了一样的问题，我觉得应该不是端口的原因，因为我的sunshine和moonlight局域网串流的时候，电脑接入以太网是能正常识别并串流的

**Comment by Qingzhong125** [comment]
请问这个需求什么时候可以提上议事日程？PC特别是台式机还是希望有线入网。

**Comment by Yeo0o** [comment]
我也是这个问题。设置里更改了两边的端口，把端口都改成81。就ok了。

**Comment by bladeofgod** [comment]
same here!

**Comment by dafeng1002** [comment]
2024/10/28
我手机安卓、wifi连路由器，笔记本win10、wifi连接，台式机win11、有线连接路由器
遇到的情况是：手机可以发现笔记本、台式机，笔记本、台式机都能发现手机；笔记本、台式机都不能发现对方，怎么解决？

**Reply by dafeng1002** [reply]
已解决：在windows11的电脑中下载exe格式安装
